.github/workflows/tools-test.yml

```diff
@@ -24,10 +24,13 @@ jobs:
         python -m pip install --upgrade pip
         python -m pip install tox
         python -m pip install kconfiglib
+        sudo apt-get install gcc-multilib
```

Comment:
Doesn't seem to be needed for testing the new classes (pure Python).

Reply:
The test is compiling `examples/hello-world` to test if the feature really work. Sure I can mock that up, but that can get stale quickly and wouldn't test if everything is working right.